tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -1083,6 +1090,12 @@ void PipelineDumper::updateHashForPipelineOptions(const PipelineOptions *options
   hasher->Update(options->extendedRobustness.robustBufferAccess);
   hasher->Update(options->extendedRobustness.robustImageAccess);
   hasher->Update(options->extendedRobustness.nullDescriptor);
+
```

Comment:
And we have a reserved field `reserved1f`. I think if it is not involved in hash updating. We should make a assert here:
`assert(options.reserved1f == false)`

Reply:
done